package.json

```diff
@@ -29,7 +29,7 @@
     "queryResource": "query",
     "okapiInterfaces": {
       "cancellation-reason-storage": "1.1",
-      "circulation": "13.0",
+      "circulation": "14.0",
```

Comment:
Dropping support for an interface is a breaking change. You must also change the module version on line 3 to `9.0.0`.